cypress/integration/registration.spec.js

```diff
@@ -156,44 +178,71 @@ context('Actions', () => {
   });
 
   it('uses licenseWizard', () => {
-    const unpublishedTestPost = mockMyResources[1];
-    cy.visit(`/editresource/${unpublishedTestPost.identifier}]`);
+    const testLink = 'http://www.test.com';
+    cy.get('[data-testid=new-registration-link]').click();
+    cy.get('[data-testid=new-resource-link]').click();
+    cy.get('[data-testid=new-resource-link-input]').type(testLink);
+    cy.get('[data-testid=new-resource-link-submit-button]').click();
 
     cy.get('[data-testid=step-navigation-3]').click();
-    cy.get('[data-testid=licence-field]').contains('CC BY 4.0');
+    cy.get('[data-testid=licence-field] input').should('have.value', '');
 
-    cy.get('[data-testid=resource_restriction_option_ntnu-internt]').click();
+    cy.get('[data-testid=resource-restriction-option-ntnu-internt]').click();
+    cy.get('[data-testid=licence-field] input').should('have.value', 'd56b161e-05d0-45c9-b96b-5c0b37b952b4');
     cy.get('[data-testid=licence-field]').contains('ntnu-internt');
-    cy.get('[data-testid=resource_restriction_option_yes]').click();
+    cy.get('[data-testid=resource-restriction-option-yes]').click();
     cy.get('[data-testid=licence-field]').contains('CC BY-NC-ND 4.0');
-    cy.get('[data-testid=resource_restriction_option_CC_BY_4_0]').click();
+    cy.get('[data-testid=resource-restriction-option-CC_BY_4_0]').click();
     cy.get('[data-testid=licence-field]').contains('CC BY 4.0');
-    cy.get('[data-testid=resource_restriction_option_yes]').click();
+    cy.get('[data-testid=resource-restriction-option-yes]').click();
     cy.get('[data-testid=licence-field]').contains('CC BY-NC-ND 4.0');
 
-    cy.get('[data-testid=commercial_use_option_yes]').click();
+    cy.get('[data-testid=commercial-use-option-yes]').click();
     cy.get('[data-testid=licence-field]').contains('CC BY 4.0');
-    cy.get('[data-testid=commercial_use_option_NC]').click();
+    cy.get('[data-testid=commercial-use-option-NC]').click();
     cy.get('[data-testid=licence-field]').contains('CC BY-NC 4.0');
```

Comment:
Er ikke det bedre hvis dette trinnet kun sjekker om feltet inneholder 'NC'? Er ikke dette kun sjekk av at man har huket av for ikke-kommersiel radio-boks fungerer?

Reply:
Slik at hvis man senere vil omstrukturere testene?

Reply:
enig. will do